packages/openchs-android/android/app/src/debug/AndroidManifest.xml

```diff
@@ -4,5 +4,5 @@
 
     <uses-permission android:name="android.permission.SYSTEM_ALERT_WINDOW"/>
 
-    <application android:usesCleartextTraffic="true" tools:targetApi="33" tools:ignore="GoogleAppIndexingWarning" />
+    <application android:usesCleartextTraffic="true" tools:targetApi="33" tools:ignore="GoogleAppIndexingWarning" tools:replace="android:usesCleartextTraffic" />
```

Comment:
Is this change needed here as cleartextTraffic is still allowed on debug mode ?

Reply:
yeah. its `tools:replace="android:usesCleartextTraffic"` to override the usesCleartextTraffic to true irrespective of what’s coming in dependency

Reply:
we have android:usesCleartextTraffic declared inside our abha dependency as well. its giving me error in 
make run if I ignore it